mobile/search.go

```diff
@@ -1,56 +1,72 @@
 package mobile
 
 import (
-	"github.com/golang/protobuf/proto"
+	"github.com/segmentio/ksuid"
 	"github.com/textileio/textile-go/broadcast"
 	"github.com/textileio/textile-go/pb"
 )
 
-// CancelFn is used to cancel an async request
-type CancelFn struct {
+// SearchHandle is used to cancel an async search request
+type SearchHandle struct {
```

Comment:
`CancelFn` -> `SearchHandle`, now contains an `Id` which can be used to distinguish results from different searches that arrive over the event bus.